vic/drivers/cesm/src/get_global_param.c

```diff
@@ -72,6 +72,10 @@ get_global_param(FILE *gp)
                 sscanf(cmdstr, "%*s %s", flgstr);
                 global_param.time_units = str_to_timeunits(flgstr);
             }
+	    else if (strcasecmp("FULL_ENERGY", optstr) == 0) {
+		sscanf(cmdstr, "%*s %s", flgstr);
+		options.FULL_ENERGY = str_to_bool(flgstr);
+	    }
```

Comment:
I think that it was not read in as part of the CESM driver, because coupled model simulations ONLY make sense with `options.FULL_ENERGY` set to `TRUE`. However, you could imagine a water balance run with a data atmosphere, so this would be OK. Since that is a special case for the CESM driver, it would be good to print out a warning if `options.FULL_ENERGY` is set to `FALSE`.

So I suggest changing:
```C
            else if (strcasecmp("FULL_ENERGY", optstr) == 0) {
                sscanf(cmdstr, "%*s %s", flgstr);
                options.FULL_ENERGY = str_to_bool(flgstr);
            }
```
to
``` C
            else if (strcasecmp("FULL_ENERGY", optstr) == 0) {
                sscanf(cmdstr, "%*s %s", flgstr);
                options.FULL_ENERGY = str_to_bool(flgstr);
                if (options.FULL_ENERGY == FALSE) {
                    log_warn("FULL_ENERGY is set to FALSE. Make sure that this is intentional.");
                }
            }
```